engine-tests/src/main/java/org/terasology/TerasologyTestingEnvironment.java

```diff
@@ -71,9 +56,9 @@
 
     @BeforeAll
     public static void setupEnvironment() throws Exception {
-        final JavaArchive homeArchive = ShrinkWrap.create(JavaArchive.class);
-        final FileSystem vfs = ShrinkWrapFileSystems.newFileSystem(homeArchive);
-        PathManager.getInstance().useOverrideHomePath(vfs.getPath(""));
+        Path tempHome = Files.createTempDirectory("terasology-tte", (FileAttribute<?>[]) null);
```

Comment:
Would [`@TempDir`](https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory).work here too? I guess if the `@BeforeAll` annotation works here, `@TempDir` should too.